banzai/qc/pattern_noise.py

```diff
@@ -58,13 +69,32 @@ def check_for_pattern_noise(self, data):
         trimmed_data = trim_image_edges(data)
         power_2d = get_2d_power_band(trimmed_data)
         snr = compute_snr(power_2d)
-        convolved_snr = convolve_snr_with_wavelet(snr)
-        peak_maxima, std_maxima = get_peak_parameters(convolved_snr)
+        fraction_pixels_above_threshold = self.get_n_grouped_pixels_above_threshold(snr) / len(snr)
```

Comment:
In Python 2, this will do integer division.

Reply:
I've updated to cast the denominator to a float.